src/core/pal/feature.cpp

```diff
@@ -1793,10 +1795,16 @@ std::vector< std::unique_ptr< LabelPosition > > FeaturePart::createCandidates( P
           case QgsPalLayerSettings::OverPoint:
```

Comment:
Can you split up this case statement to separate the Around/Over modes now? the amount of shared logic is much less than the specific logic for these two options